### PR TITLE
Add new CUDA versions.

### PR DESCRIPTION
**Pull request by jrhemstad**
Add CUDA versions: 11.1.1, 11.2.0, 11.2.1, 11.2.2

See also: https://github.com/compiler-explorer/infra/pull/487

### CODE REVIEW

File: etc/config/cuda.amazon.properties [hunk 1]
```diff
@@ -39,9 +39,21 @@ compiler.nvcc11.semver=11.0.2
 compiler.nvcc11u1.exe=/opt/compiler-explorer/cuda/11.0.3/bin/nvcc
 compiler.nvcc11u1.options=--compiler-bindir /opt/compiler-explorer/gcc-9.3.0/bin
 compiler.nvcc11u1.semver=11.0.3
-compiler.nvc111.exe=/opt/compiler-explorer/cuda/11.1.0/bin/nvcc
-compiler.nvc111.options=--compiler-bindir /opt/compiler-explorer/gcc-10.2.0/bin
-compiler.nvc111.semver=11.1.0
+compiler.nvcc111.exe=/opt/compiler-explorer/cuda/11.1.0/bin/nvcc
+compiler.nvcc111.options=--compiler-bindir /opt/compiler-explorer/gcc-10.2.0/bin
+compiler.nvcc111.semver=11.1.0
```
Comment (partouf) [inline etc/config/cuda.amazon.properties:44]: this will need an extra `compiler.nvcc111.alias=nvc111` in case it was ever used in a shortlink

Comment (jrhemstad) [reply etc/config/cuda.amazon.properties:44]: I don't quite understand. What's different about this one that it needs the alias? Do the other ones need it too?

Comment (partouf) [reply etc/config/cuda.amazon.properties:44]: this one was renamed from nvc111 to nvcc111

Comment (partouf) [reply etc/config/cuda.amazon.properties:44]: other ones do not need it, just existing id's that are changed

**Comment by partouf** [comment]
These are now live, thanks!

**Pull request merged**